application/src/main/java/org/opentripplanner/osm/model/OsmWithTags.java

```diff
@@ -11,7 +11,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
-import javax.annotation.Nullable;
+import org.jetbrains.annotations.Nullable;
```

Comment:
Was this intentional?

Reply:
sorry, don't know why my IDE is changing it and can't find the Nullable in `javax.annotation`